src/native/windows/apps/prunsrv/prunsrv.c

```diff
@@ -258,6 +259,8 @@ static BOOL   gSignalValid   = TRUE;
 static APXJAVA_THREADARGS gRargs;
 static APXJAVA_THREADARGS gSargs;
 
+static DWORD stopStarted = 0; /* Not stop not started */
```

Comment:
Comment looks odd. Does it mean the time that the stop process began? A rename might be helpful here. Maybe `stopCalledTime`?

Reply:
fixed.